data/json/items/armor/gloves.json

```diff
@@ -1161,7 +1161,7 @@
     "id": "lc_demi_gaunt",
     "type": "ARMOR",
     "category": "armor",
-    "name": { "str_sp": "mild steel demi-gauntlet" },
+    "name": { "str": "mild steel demi-gauntlet", "str_pl": "pairs of mild steel demi-gauntlets" },
```

Comment:
Should the singulars for most of these be gauntlets or even pair of [type] gauntlets?

Reply:
Good catch, it seems it's the case for other gauntlets, let me fix that.